create-react-app/my-app/src/components/Photos.jsx

```diff
@@ -3,11 +3,11 @@ import Photo from '../components/Photo'
 
 const Photos = (props) => {
 	let getPhotos = props.photos.map(photo => (
-	  <Photo 
-	  	key={photo.id} 
-	  	src={photo.thumbnailUrl} 
+	  <Photo
+	  	key={photo.id}
+	  	src={photo.thumbnailUrl}
 	  	alt={photo.title}
-	  	onClick={props.showPhoto.bind(null, photo)}
```

Comment:
@emmareardon1 the onClick needs to be applied to the HTML element, not to the component. Check the create-react-app/my-app/src/components/Photo.jsx line 5